gcsio/src/main/java/com/google/cloud/hadoop/gcsio/ListObjectOptions.java

```diff
@@ -42,6 +43,9 @@ public static Builder builder() {
   @Nullable
   public abstract String getDelimiter();
 
+  /** Whether to include prefix object in the result. */
+  public abstract boolean isIncludePrefix();
```

Comment:
would isIncludePrefixEnabled be a better name? isIncludeNoun sounds not right.

Reply:
Yeah, I don't like `Enabled` suffixes that we have in some places, I guess better name will be `shouldIncludePrefix` but it's not supporter by AutoValue. I can try to drop `is` prefix from getter, if it works?

Reply:
sounds good.